cmd/sovereignnode/systemTestDemo/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/urfave/cli"
+)
+
+var (
+	log = logger.GetOrCreate("checker")
+)
+
+const (
+	proxyPem          = "scripts/testnet/testnet-local/sandbox/proxy/config/walletKey.pem"
+	subscribedAddress = "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"
```

Comment:
Yes, should be the same. Could be somehow reused, but would require more code-work, which I wanted to avoid. This is related to my PR description from here:
> This code(go) shall be reviewed as a testing framework code, not as a productive code. This is the reason why many values are hard-coded and certain variables are expected to be set to a specific value